playbooks/roles/ocp-cost-management/tasks/cmo-test.yml

```diff
@@ -0,0 +1,112 @@
+---
+- name: Setup and Test Cost Management Operator in OpenShift
+  hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: Log in to OpenShift

```

Comment:
this task is not required as we have specified the KUBECONFIG to be used in vars folder